src/browser/index.tsx

```diff
@@ -24,4 +24,19 @@ import AppInit, { setupSentry } from './AppInit'
 import './init'
 
 setupSentry()
-ReactDOM.render(<AppInit />, document.getElementById('mount'))
+;(async () => {
+  const doesPreferQuery = localStorage.getItem('prefersOldBrowser') === 'false'
+
+  const response = await fetch('/preview/manifest.json')
+  if (response.status === 200) {
+    if (doesPreferQuery) {
+      window.location.pathname = '/preview/'
```

Comment:
In the bundled Browser case, the preview will be on `http://localhost:7474/browser/preview` and in the hosted case it's on `browser.neo4j.io/preview`.

So this would not work for bundled browser